<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>queryDialog</class>
 <widget class="QDialog" name="queryDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: transparent;
background-color: rgb(248, 248, 255);</string>
  </property>
  <widget class="QPushButton" name="but_close">
   <property name="geometry">
    <rect>
     <x>1329</x>
     <y>0</y>
     <width>37</width>
     <height>37</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icon/close.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="but_previous">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>720</y>
     <width>56</width>
     <height>34</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icon/left.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_79">
   <property name="geometry">
    <rect>
     <x>672</x>
     <y>720</y>
     <width>16</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;黑体&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_N">
   <property name="geometry">
    <rect>
     <x>603</x>
     <y>720</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 24pt &quot;黑体&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="but_last">
   <property name="geometry">
    <rect>
     <x>1200</x>
     <y>720</y>
     <width>56</width>
     <height>34</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icon/end.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_M">
   <property name="geometry">
    <rect>
     <x>684</x>
     <y>724</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(0, 0, 0);</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>1281</width>
     <height>501</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView{
    font: 14pt &quot;黑体&quot;; 
	border-width:1px;  
	border-style:solid;
	border-color: rgb(51, 153, 255);  
	background-color: rgb(248, 248, 255);
}
QTableView::item:seleted{
    background-color: rgb(248, 248, 255); 
	color: black;
}
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="but_first">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>720</y>
     <width>56</width>
     <height>34</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icon/home.bmp);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="but_next">
   <property name="geometry">
    <rect>
     <x>893</x>
     <y>720</y>
     <width>56</width>
     <height>34</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/icon/right.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(248, 248, 255);
border-style:solid;
background-color: rgb(51, 153, 255);</string>
   </property>
   <property name="text">
    <string>车辆信息查询</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>1281</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    font: 14pt &quot;黑体&quot;;
    margin-top: 1ex;
    border-width:0.5px;  
    border-style:solid;
    border-color: rgb(51, 153, 255);  
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left:15px;      
    margin-left: 5px;
    margin-right: 5px;
    padding:1px;
}</string>
   </property>
   <property name="title">
    <string>查询条件选择</string>
   </property>
   <widget class="QPushButton" name="but_search">
    <property name="geometry">
     <rect>
      <x>1158</x>
      <y>18</y>
      <width>115</width>
      <height>115</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: solid;
background-color: rgb(51, 153, 255);
image: url(:/icon/search.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>57</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>最新</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>57</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>条</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="limited">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;
border-color:rgb(51, 153, 255); 
border-style:solid;
border-top-width:0px;
border-right-width:0px;
border-bottom-width:2px;
border-left-width:0px;</string>
    </property>
    <property name="text">
     <string>10000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_IN">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox{
   font: 14pt &quot;黑体&quot;;
   spacing: 10 
}  

QCheckBox::indicator {
       width: 25px;
       height: 25px; 
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
} 
QCheckBox::indicator:checked {
image: url(:/icon/check.png);
}
</string>
    </property>
    <property name="text">
     <string>进站</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_OUT">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox{
   font: 14pt &quot;黑体&quot;;
   spacing: 10 
}  

QCheckBox::indicator {
       width: 25px;
       height: 25px; 
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
} 
QCheckBox::indicator:checked {
image: url(:/icon/check.png);
}
</string>
    </property>
    <property name="text">
     <string>出站</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>60</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;</string>
    </property>
    <property name="text">
     <string>车型选择</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_carType">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>60</y>
      <width>200</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {
font: 14pt &quot;Arial&quot;;
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
margin-top: 0ex;
}
QComboBox::drop-down {
width:30px; 
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
}
QComboBox::down-arrow {
image: url(:/icon/arrow.png);
}
QComboBox QAbstractItemView{
height:50px;
border: 1px solid rgb(51, 153, 255); 
selection-background-color: rgb(51, 153, 255); 
}
QScrollBar:vertical{
width:30px;
border: 2px solid grey;
}
QComboBox QAbstractItemView::item{
height:20px;
}</string>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="maxVisibleItems">
     <number>10</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_L1">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>60</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox{
   font: 14pt &quot;黑体&quot;;
   spacing: 10 
}  

QCheckBox::indicator {
       width: 25px;
       height: 25px; 
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
} 
QCheckBox::indicator:checked {
image: url(:/icon/check.png);
}
</string>
    </property>
    <property name="text">
     <string>一线</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_L2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>60</y>
      <width>85</width>
      <height>30</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QCheckBox{
   font: 14pt &quot;黑体&quot;;
   spacing: 10 
}  

QCheckBox::indicator {
       width: 25px;
       height: 25px; 
border-width:1px; 
border-style:solid; 
border-color:rgb(51, 153, 255); 
} 
QCheckBox::indicator:checked {
image: url(:/icon/check.png);
}
</string>
    </property>
    <property name="text">
     <string>二线</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_147">
    <property name="geometry">
     <rect>
      <x>1085</x>
      <y>110</y>
      <width>21</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>条</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_146">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>110</y>
      <width>67</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>结 果：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_num">
    <property name="geometry">
     <rect>
      <x>1003</x>
      <y>110</y>
      <width>76</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(0, 0, 0);
border-style:solid;
border-top-width:0px;
border-right-width:0px;
border-bottom-width:2px;
border-left-width:0px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
